ethportal-api/README.md

```diff
@@ -41,7 +41,7 @@ async fn main() {
     assert!(result);
 
     // Call portal_historyLocalContent endpoint and deserialize to `HistoryContentValue::BlockHeaderWithProof` type
-    let result: HistoryContentValue = client.local_content(content_key).await.unwrap();
+    let result: PossibleHistoryContentValue = client.local_content(content_key).await.unwrap();
     assert_eq!(result, content_item);
```

Comment:
```suggestion
    assert_eq!(result, PossibleHistoryContentValue::ContentPresent(content_item.clone()));
```

Reply:
Yup, this is also needed, though it doesn't seem like the clone is necessary